src/components/AddItem.js

```diff
@@ -0,0 +1,51 @@
+import "../styles/AddItem.css";
+
+function AddItem() {
+  return (
+    <main className="main">
+      <form className="add-item-register-container">
+        <div className="form-header">
+          <h3 className="form-title">상품 등록하기</h3>
+          <button className="form-btn">등록</button>
+        </div>
+        <label htmlFor="item-img">상품 이미지</label>
+        <input
+          type="file"
+          className="file-input"
+          id="item-img"
+          placeholder="상품 이미지"
+        ></input>
```

Comment:
self closing tag 사용해볼 수 있어용! 
```suggestion
        <input
          type="file"
          className="file-input"
          id="item-img"
          placeholder="상품 이미지"
        />
```